Meta/Lagom/ReadMe.md

```diff
@@ -130,3 +130,28 @@ You may run into annoying issues with the stacktrace:
 
 That means it couldn't find the executable `llvm-symbolizer`, which could be in your OS's package `llvm`.
 `llvm-symbolizer-11` will [not be recognized](https://stackoverflow.com/a/42845444/).
+
+## Using Lagom in an External Project
+It is possible to use Lagom for your own projects outside of Serenity too\!  
```

Comment:
- Do we need to escape the `!`? Would be news to me.
- Let's not rely on trailing whitespace to insert newlines. Every sane editor will strip those on save, just insert a newline manually. Might be a `<br>` vs `<p>` difference, but who cares :^)

Reply:
For the whitespace linebreak to work, I needed to escape the `!`. The whitespace is pretty silly, so I've removed it, along with the escaping for the `!`.